content/en/docs/armory-admin/dinghy-enable.md

```diff
@@ -58,10 +58,74 @@ hal armory dinghy enable
 {{< /tabs >}}
 
 
+## Configuring SQL
+Dinghy can use Redis to store relationships between pipeline templates and pipeline dinghy files. An external MySQL instance is highly recommended for production use. If MySQL becomes unavailable, dinghy files will need to be updated in order to repopulate Redis with the relationships.
```

Comment:
@Jossuecito should line 62 say

> If MySQL becomes unavailable, dinghy files will need to be updated in order to repopulate **MySQL** with the relationships.

instead? It currently says repopulate Redis with the relationships